l10n/en/mozorg/home-m24.ftl

```diff
@@ -0,0 +1,106 @@
+m24-home-break-free-from = Break free from big tech — our products put you in control of a safer, more private internet experience.
+m24-home-explore-our-products = Explore our products
+m24-home-firefox = { -brand-name-firefox }
+m24-home-get-the-gold = Get the gold standard for browsing with speed, privacy and control.
+m24-home-thunderbird = { -brand-name-thunderbird }
+m24-home-go-chaosfree-with = Simplify your life with one app for all your emails, calendars and contacts.
+m24-home-fakespot = { -brand-name-fakespot }
+# "fake laughs" is a joke and the ;) is a winking face to hint it's a joke. You can translate this in a away that keeps both the repetition and the joke: fake reviews, fake products, fake ___. Or leave the joke and winking face out
+m24-home-spot-fake-reviews = Spot fake reviews, products and maybe one day, even fake laughs ;)
+m24-home-pocket = { -brand-name-pocket }
+m24-home-save-the-internets = Save the internet’s best content on any device — because who has time to read everything now?
+m24-home-mozilla-vpn = { -brand-name-mozilla-vpn }
+m24-home-keep-your-location = Keep your location and online adventures private — stream like a local, anywhere.
+m24-home-mozilla-monitor = { -brand-name-mozilla-monitor }
+m24-home-get-a-headsup = Get a heads-up if your personal info is at risk and lock it down like a pro.
+
+## Donate
+
+m24-home-donate-to-the = Donate to the { -brand-name-mozilla } non-profit
+m24-home-mozilla-is-building = { -brand-name-mozilla } is building a movement to reclaim the internet. Together we can build a future where our privacy is protected, AI is trustworthy and irresponsible tech companies are held accountable. But that’s only possible if we do it together.
+m24-home-were-proudly-nonprofit = We’re proudly non-profit. Will you donate to { -brand-name-mozilla } today?
+m24-home-donate = Donate
+
+## AI gallery
+
+m24-home-tag-product = Product
+m24-home-tag-program = Program
+m24-home-tag-event = Event
+m24-home-read-more = Read more
+m24-home-get-started = Get started
+m24-home-join-the-movement = Join the movement:<br> AI for the people
+m24-home-our-mission-is = Our mission is to make it easy for people to build with, and collaborate on, open-source, trustworthy AI.
+m24-home-explore-mozilla-ai = Explore { -brand-name-mozilla-ai }
+m24-home-llamafile = { -brand-name-llamafile }
+m24-home-run-large-language = Run large language models (LLMs) easily on your computer with { -brand-name-llamafile } — no installation needed, and your data stays safe on your device.
+m24-home-mozilla-builders= { -brand-name-mozilla } Builders
+m24-home-turn-your-ai = Turn your AI idea into reality with our 12-week accelerator. Get funding, mentorship and a chance to pitch your innovation to investors.
+m24-home-lumigator = { -brand-name-lumigator}
+m24-home-find-the-just = Find the “just right” LLM for your needs, use case and data — like Goldilocks, but with sharper teeth.
```

Comment:
**Question:** is Goldilocks well known throughout the world - will everyone get the joke? (I would assume yes, but I feel like I am also probably ignorant here).

Reply:
✅ It's hard to fight the urge to add the Oxford comma while I'm in here...